1. создал компонент Article

Alex-nzn opened the pull request:
2. создал папку img в src
3. создал uikit-компонент uikit-Article.jsx
4. добавил его на страницу uikit.jsx

Alex-nzn committed:
1. создал компонент Article

2. создал папку img в src
3. создал uikit-компонент uikit-Article.jsx
4. добавил его на страницу uikit.jsx

Alex-nzn committed:
Выровнял footer по вертикали

.info__item  добавил font-weight: 600;

nobleworkshop commented on src/components/Article/Article.jsx:
```diff
@@ -0,0 +1,35 @@
+import styles from "./article.module.css";
+import Badge from "../Badge/Badge";
+
```
Лишний отступ. Одной строки достаточно.

nobleworkshop commented on src/components/Article/article.module.css:
```diff
@@ -0,0 +1,53 @@
+body {
```
тег body здесь не нужен, это файл стилей только для текущего компонента.

nobleworkshop commented on src/components/Article/article.module.css:
```diff
@@ -0,0 +1,53 @@
+}
+
+.article {
+    width: 1055px;
```
Убрать фикс ширину и высоту. Сделать компонент адаптивным под мобильные устройства.

nobleworkshop commented on src/components/Article/Article.jsx:
```diff
@@ -0,0 +1,35 @@
+                    </div>
+
+                    <div>
+                        <Badge title="Badge" />
```
Badge сейчас прописан фиксированно, и будет одинаковым для всех статей созданных этим компонентом.

 Сделать так чтобы компонент принимал себя Props с именем категории и бэйдж выводил имя категории. Также надо будет подумать и реализовать ссылку на категорию.

nobleworkshop commented on src/components/Article/Article.jsx:
```diff
@@ -0,0 +1,35 @@
+                    <p className={styles.text}>{text}</p>
+                </div>
+                <div className={styles.footer}>
+                    <div className={styles.info}>
```
Для инфо линии (автор, дата, timeToRead) уже сделан отдельный компонент. Давайте переиспользуем его когда он будет влит в develop.

Также проследить чтобы имена пропсов здесь были следующие: author, date, timeToRead.

nobleworkshop reviewed (changes requested):
Хорошо 👍, есть пару правок.

Alex-nzn committed:
Всё исправил, Шэф
1. убрал лишний отступ Article.jsx
2.  убрал тег body  в  article.module.css
3. убрал ширину и высоту у .article
4. сделал адаптацию под мобилку
5. добавил пропсы к Badge (имя, ссылку)
6. создал страницу art для Badge
7. заменил имена пропсов блока .info на рекомендованные

nobleworkshop reviewed (commented):
Отлично!) 

Создавать страницу art под ссылку с беёджа не обязательно, пока делаем только главную. Ссылки делаем, но они пока остаются не рабочими. 

Стоит еще поменять общий тег статьи на article.

И внедрить компонент detailsitem, он уже есть в develop. Как получить его в ветке article - написал в чате. Дублирую:

> Нужно стянуть изменения из develop в ветку article
> 
> Для этого надо сделать merge из develop в article.
> 
> После в ветке article появится компонент detailsItem, использоватиь его в компоненте со статьёй.

Alex-nzn committed:
Усё готово

страницу art не стал удалять
общий тег заменил на article
внедрил компонент DetailsItem

Alex-nzn committed:
Собрал Header

Alex-nzn committed:
Изменил описание подключения в uikit-Header

nobleworkshop commented on src/uikit-components/uikit-Article.jsx:
```diff
@@ -0,0 +1,44 @@
+                        {`<Article    img={img}
+            title="Hope dies last"
+            text="Lorem ipsum ..."
+            info_text="Jakob Gronberg"
```
Пофиксить пример вызова компонента в документации. Сейчас тут старые имена пропсов.

nobleworkshop reviewed (approved):
Отлично 👍. 1 фикс и можно мержить.

Пофиксить пример вызова компонента в документации. Сейчас тут старые имена пропсов.

Alex-nzn committed:
Заменил имена пропсов в описании ukit-Article на актуальные

nobleworkshop reviewed (approved):
Можно мержить 👍 ✅

Alex-nzn committed:
Merge branch 'develop' into article
